rollup.config.js

```diff
@@ -20,7 +20,7 @@ export default packages.reduce((config, packageName) => {
       useTsconfigDeclarationDir: true,
       clean: true,
       verbosity: 1,
-      tsconfig: `${root}/tsconfig.json`,
+      tsconfig: `${root}/_tsconfig.json`,
```

Comment:
I'm thinking about removing all `_tsconfig.json` files and adding:
```typescript
tsconfigOverride: {
  include: [`${root}/src/`],
  compilerOptions: {
    baseUrl: root,
    declarationDir: `${root}/dist/types`,
  },
},
```
below. 

I think that this might be a cleaner approach...